roles/nova-data/tasks/libvirt.yml

```diff
@@ -57,28 +57,34 @@
   when: openstack_install_method == 'source'
 
 - name: update various lines in libvirtd.conf
-  lineinfile: dest=/etc/libvirt/libvirtd.conf regexp="{{ item.value.regexp }}"
-              line="{{ item.value.line }}"
+  lineinfile:
+    dest: /etc/libvirt/libvirtd.conf
+    regexp: "{{ item.value.regexp }}"
+    line: "{{ item.value.line }}"
   with_dict:
-    ltcp:
+    listen_tcp:
       regexp: '^listen_tcp\s*='
       line: "listen_tcp = 1"
```

Comment:
Lets turn off listen_tcp. It got originally enabled to support kvm live migration, but we'll be doing it over ssh, and not doing block migration. I think we can still do resize and live migration without tcp listening. (and we probably don't want to listen without TLS)